cognee/modules/ingestion/identify.py

```diff
@@ -1,7 +1,9 @@
 from uuid import uuid5, NAMESPACE_OID
 from .data_types import IngestionData
 
-def identify(data: IngestionData) -> str:
-    data_id: str = data.get_identifier()
+from cognee.modules.users.models import User
 
-    return uuid5(NAMESPACE_OID, data_id)
+def identify(data: IngestionData, user: User) -> str:
+    data_content_hash: str = data.get_identifier()
+    # return UUID hash of file contents + owner id
+    return uuid5(NAMESPACE_OID,f"{data_content_hash}{user.id}")
```

Comment:
Some space is missing.

Reply:
I would say we need owner_id for permissions regarding some Data operations as they are no longer tied to Datasets. 

content_hash we don't need right now, but might be nice to have as it's the only way to verify the content of the Data